docs/modules/tiered-storage/pages/performance.adoc

```diff
@@ -0,0 +1,53 @@
+= Performance
+
+== Throughput vs Access Pattern
+
+Tiered Storage is sensitive to disk reads and therefore performs best when operating with entries retrieved mostly from the memory.
+The higher the probability of hitting an entry in the memory, the closer Tiered Storage gets to the throughput of HD.
+This means random access pattern is the worst for Tiered Storage, while predictable access patterns working mostly with the hot cache fitting in the memory is the best.
+
+== Impact of the Access Latency of the Devices
+
+Tiered Storage reads from the device synchronously, making the access latency of the device directly translating to throughput when the device needs to be touched.
+This rules out all high latency devices (HDDs, SAN drives, cloud block storages etc) from the list of the recommended types.
+
+== Compaction is the Driver of the Performance
+
+Compaction runs synchronously before/after the IMap operations making it the main driver/limiter of the performance of Tiered Storage.
+There are a few aspects that determine the impact of compaction on the performance of Tiered Storage:
+
+- *Frequency of the compaction*: The less frequently it runs, the less impact it has on the latency, therefore, on the throughput.
+The compactor gets active when the used space or the amount of garbage reaches the configured threshold.
+Then it runs incrementally and time-boxed to make the smallest impact possible on latency.
+
+- *Efficiency of the compaction*: The compaction is the most efficient when it can just clean up disk space without relocating alive entries inside the address space of Tiered Storage.
+This can be impacted by trading off disk space for compaction performance, see later.
+
+- *Hot cache thrashing*: The compactor Tiered Storage employs relocates the alive entries it encounters with to the memory where the hot cache resides.
```

Comment:
"employs relocates"?

Reply:
Rephrased in https://github.com/hazelcast/hz-docs/pull/1047/commits/0253a16cd6c5ede529c5f0f74b0e730fe62b5d95